code/game/gamemodes/game_mode.dm

```diff
@@ -118,6 +118,9 @@
 	if(report)
 		addtimer(CALLBACK(src, .proc/send_intercept, 0), rand(waittime_l, waittime_h))
 	generate_station_goals()
+
+	addtimer(CALLBACK(SSabnormality_queue, /datum/controller/subsystem/abnormality_queue/proc/HandleStartingAbnormalities), ABNORMALITY_DELAY)
```

Comment:
That's a very funny way to do it, I was expecting you to put it into master controller.

Reply:
Touching master controller is one of the absolute worst things you can do, subsystems exist for a reason
Coincidentally this is also how SSdbcore does its things just a few lines above